chacra/tests/controllers/repos/test_repos.py

```diff
@@ -13,7 +13,7 @@ def test_repo_exists(self, session):
         )
         repo.path = "some_path"
         session.commit()
-        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/api/')
+        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/')
```

Comment:
I was thinking that we wanted `api/` to provide something aside from the normal repo directories, but since that is no longer the case because the actual repo will live elsewhere, then I took it out. Hope that is OK?